kids-chatbot/src/components/SubscriptionPlans.tsx

```
@@ -0,0 +1,138 @@
+                {formatPrice(plan.price)} تومان
+              </div>
+              <div className="text-sm text-gray-500">
+                {plan.duration} روز
+              </div>
+            </div>
+
+            <div className="space-y-3 mb-6">
+              {plan.features.map((feature, index) => (
+                <div key={index} className="flex items-center">
+                  <svg className="w-5 h-5 text-green-500 ml-3" fill="none" stroke="currentColor" viewBox="0 0 24 24">
+                    <path strokeLinecap="round" strokeLinejoin="round" strokeWidth={2} d="M5 13l4 4L19 7" />
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)